hack/update-all.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+hack/update-codegen.sh
+hack/update-test-codegen.sh
```

Comment:
@aylei 
hi, I renamed the `hack/test-codegen.sh` to `hack/update-test-codegen.sh`  to follow the update script patterh `hack/update-xxx.<ext>`.  I think it's better to name the same type of scripts consistently.

Reply:
Thanks!